_source/logzio_collections/_log-sources/azure-blob-trigger.md

```diff
@@ -0,0 +1,208 @@
+
+Click **Go to resource group** to go to your resource group with all the created resources.
+
+
+##### Check Logz.io for your logs
+
+Give your logs some time to get from your system to ours, and then open [Kibana](https://app.logz.io/#/dashboard/kibana/discover?). You can filter for logs of `type` `blobStorage` to see the incoming logs.
+
+If you still don’t see your logs, see [log shipping troubleshooting](https://docs.logz.io/user-guide/log-shipping/log-shipping-troubleshooting.html).
+
+
+
+</div>
+</div>
+<!-- tab:end -->
+
+<!-- tab:start -->
+<div id="existing-blob-config">
+
+
+
+#### Connect to existing blob storage account
+
+**Before you begin, you'll need**: a blob storage account of the type **StorageV2 (general purpose v2)**.
+
+
+<!-- info-box-start:info -->
+If your existing blob storage account is of any other kind, it will NOT work. Instead, follow the process to set up a new blob storage account.
+{:.info-box.important}
+<!-- info-box-end -->
+
+
+<div class="tasklist">
+
+##### Check your storage account for compatibility
+
+Double-check your [_Storage accounts_](https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.Storage%2FStorageAccounts) to make sure that they are compatible with this integration. They should be of the type **StorageV2 (general purpose v2)**.
+
+
+##### Launch an automated deployment
+
+👇 Click this button:
+
+[![Deploy to Azure](https://dytvr9ot2sszz.cloudfront.net/logz-docs/azure_blob/deploybutton-az.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Flogzio%2Flogzio-azure-blob-trigger%2Fmain%2Fazure%2Ffunction-auto-deployment.json)
+{:.override.btn-img}
+  
+
+##### Fill in the deployment parameters
+
+In the **Custom deployment** screen, fill in all the parameters as per table below and click **Review + create**.
+
+![Custom deployment](https://dytvr9ot2sszz.cloudfront.net/logz-docs/azure_blob/blob-trigger-1.png)
+
+| Parameter | Description |
+| --- | --- |
+| Storage Account Name | The storage account (general purpose v2) name. |
+| Container Name | The name of the container inside the storage account |
+| Logs Path | The path from where blob files will trigger the Logz.io function (including sub-directories in that path). Leave empty if you want every blob file in the container to trigger the Logz.io function. |
+| Logzio URL | {% include log-shipping/listener-var.md %} |
+| Logzio Token | {% include log-shipping/log-shipping-token.md %} |
+| Multiline Regex | The regex that matches the multiline logs in text blob files. Leave empty if you do not use multiline logs in your text blob files. |
+
+<!-- info-box-start:info -->
+Only new logs that are created from the moment the integration process is complete are sent to Logz.io. Logs that were added before this integration are not sent to Logz.io.
+{:.info-box.important}
+<!-- info-box-end -->
+
```

Comment:
This integration does the exact opposite :)
Please change this info to:
Logs that were in the container before the deployment will be shipped to Logz.io. If these logs have already been shipped to Logz.io, we recommend that you empty the container before the deployment.

You can also put the information at the top of the page after the other info.